packages/adapters/fastify/web/src/web.ts

```diff
@@ -70,14 +70,7 @@ export async function redwoodFastifyWeb(
       })
     }
 
-    // Make sure apiUrl starts and ends with a slash
-    const prefix = redwoodOptions.apiUrl.charAt(0) !== '/' ? '/' : ''
-    const suffix =
-      redwoodOptions.apiUrl.charAt(redwoodOptions.apiUrl.length - 1) !== '/'
-        ? '/'
-        : ''
-
-    const apiUrlWarningPath = `${prefix}${redwoodOptions.apiUrl}${suffix}`
+    const apiUrlWarningPath = coerceRootPath(redwoodOptions.apiUrl)
```

Comment:
We use the same logic in this file to correctly register the warning route discussed as a friction point, so using the function here